storybook/storybook-react/src/Grid/Grid.stories.tsx

```diff
@@ -3,25 +3,62 @@
  * Copyright (c) 2023 Gemeente Amsterdam
  */
 
-import { Grid, Image, Screen } from '@amsterdam/design-system-react'
+import type { GridCellProps, GridProps } from '@amsterdam/design-system-react'
+import { Grid, Screen } from '@amsterdam/design-system-react'
 import { Meta, StoryObj } from '@storybook/react'
 import { paddingArgType } from '../shared/argTypes'
 
+type StoryProps = GridProps & GridCellProps
+
+const argTypes = {
+  children: {
+    table: { disable: true },
+  },
+}
+
+const gridArgTypes = {
+  compact: {
+    control: { type: 'boolean' },
+  },
+  gapVertical: {
+    control: {
+      type: 'radio',
+      labels: { small: 'small', undefined: 'medium', large: 'large' },
+    },
+    options: ['small', undefined, 'large'],
+  },
+  paddingVertical: paddingArgType,
+  paddingTop: paddingArgType,
+  paddingBottom: paddingArgType,
+}
+
+const gridCellArgTypes = {
+  span: {
+    control: { type: 'inline-radio' },
+    options: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12],
+  },
+  start: {
+    control: { type: 'inline-radio' },
+    options: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12],
+  },
```

Comment:
Wouldn't number inputs make more sense here?

Reply:
Not sure. A number input becomes full width and requires typing or clicking very small buttons. I think radio options are easier to use, even here.

Reply:
A number input allows you to use the arrows on your keyboard though, which makes them easier to use I think